src/lib/models/appMap.js

```diff
@@ -61,23 +61,6 @@ export default class AppMap {
     delete this.data.classMap;
   }
 
-  /* eslint-disable class-methods-use-this */
-  get(fqid) {
-    const result = {
-      type: null,
-      object: null,
-    };
-
-    const fqidMatch = fqid.match(/^([a-z]+):(.+)/);
-    if (fqidMatch) {
-      result.type = fqidMatch[1]; // eslint-disable-line prefer-destructuring
-      result.object = fqidMatch[2]; // eslint-disable-line prefer-destructuring
-    }
-
-    return result;
-  }
```

Comment:
After thinking about it, this doesn't really make sense because there's no first-class support for labels as objects (similar to an event or code object).